.gitignore

```diff
@@ -0,0 +1,94 @@
+/omod/target
```

Comment:
@slubwama Add a template for Visual Studio Code

Reply:
@ssmusoke I have no Idea how that looks like since I don't use it. Send it via slack or you will add that when you fork master when this is merged

Reply:
@slubwama Just right click in the .ignore file and select Add Template 
<img width="476" alt="Screenshot 2019-07-03 15 08 43" src="https://user-images.githubusercontent.com/689900/60590426-99ad9c80-9da4-11e9-854a-146f6fbc7bfb.png">

Reply:
@ssmusoke  should we be Commit the ignore file??

Reply:
Yes .gitignore files are always committed

Reply:
@ssmusoke what if someone used eclipse and netbins what happens. do we commit all that. to me that doesn't make sense

Reply:
@slubwama That is why there are templates for the most common IDEs and language specific needs - you can leave out Eclipse & Netbeans